src/intents.cs

```diff
@@ -509,7 +524,9 @@ public enum INRequestRideIntentResponseCode : long {
 		FailureRequiringAppLaunchMustVerifyCredentials,
 		FailureRequiringAppLaunchNoServiceInArea,
 		FailureRequiringAppLaunchServiceTemporarilyUnavailable,
-		FailureRequiringAppLaunchPreviousRideNeedsCompletion
+		FailureRequiringAppLaunchPreviousRideNeedsCompletion,
+		[iOS (13,0), Watch (6,0)]
+		FailureRequiringAppLaunchRideScheduledTooFar,
```

Comment:
I would normally agree but this one in particular does need it since it is not something you get from an API but a value that you actually provide to it.

```
[Export ("initWithCode:userActivity:")]
[DesignatedInitializer]
IntPtr Constructor (INRequestRideIntentResponseCode code, [NullAllowed] NSUserActivity userActivity);
```

I think we should keep the availability attributes.